src/components/user-document-rating-dialog.yml

```diff
@@ -0,0 +1,31 @@
+userDocumentRatingDialog:
+  title:
+    en: Rate document
+    de: Dokument bewerten
+  noExistingRatingExplanationMarkdown:
+    en: |
+      You have not yet submitted a rating for this document.
```

Comment:
Small reminder: we could drop the "yet" / "noch" to cover the "rated, then deleted" scenario.

Reply:
Thanks, it's deleted :+1: